openapi.yaml

```
@@ -1258,13 +1258,15 @@ components:
             - "building.housingsNumber"
           properties:
             "building.housingsNumber":
-              allOf:
-                - $ref: "#/components/schemas/Issue"
               required:
                 - value
               properties:
                 value:
                   type: integer
+                  description: |
+                    Nombre de logement de l'immeuble.
+
+                    Ce champ est nommé `NombreLogementsAdresseIPE` dans le fichier IPE.
```

Au final, le paramètre "building.housingsNumber" ne contient qu'un seul champ "value", donc autant supprimer ce champ et considéré "building.housingsNumber" comme la valeur.
De plus, on peut rajouter "minimum: 0" pour interdire les entiers négatifs

```suggestion
              type: integer
              minimum: 0
              description: |
                Nombre de logement de l'immeuble.

                Ce champ est nommé `NombreLogementsAdresseIPE` dans le fichier IPE.
```

---

> Au final, le paramètre "building.housingsNumber" ne contient qu'un seul champ "value", donc autant supprimer ce champ et considéré "building.housingsNumber" comme la valeur. De plus, on peut rajouter "minimum: 0" pour interdire les entiers négatifs

Il est préférable de garder la structure en objet pour garder une harmonie structurel et pouvoir facilement faire évoluer le modèle si besoin dans l'avenir.

Par contre, je vais ajouter la contrainte `minimum: 0`.